Assets/Scripts/Managers/HighlightManager.cs

```diff
@@ -131,6 +148,29 @@ private void ResetSelection(GameState gameState)
                 }
 
             }
+            // temp implementation
+            if (selectedPlayer == null)
+            {
+                return;
+            } else
+            {
+                selectedPlayer.UnRenderHand();
+            }
+        } 
+    }
+
+    // This method is actually redundant since you would HAVE to see the updated deck if you have a player selected.
+    public static bool RenderHandIfAppropriate(PlayerClass player)
```

Comment:
Why bother introducing a function that is redundant? If this doesn't serve a purpose, remove it.

Reply:
"extensibility" "foresight" 
also makes the code easier to read: otherwise you would have to suss out the logic in the comment